pkg/k8shandler/reconciler.go

```diff
@@ -35,20 +34,27 @@ func Reconcile(requestCluster *logging.ClusterLogging, requestClient client.Clie
 		clusterLoggingRequest.ForwarderSpec = forwarder.Spec
 	}
 
+	clusterLoggingRequest.Cluster.Status.Conditions.SetCondition(logging.NewCondition("ConditionUnknown",
+		corev1.ConditionUnknown, "Processing", "Reconciliation in progress"))
 	if clusterLoggingRequest.IncludesManagedStorage() {
 		// Reconcile certs
 		if err = clusterLoggingRequest.CreateOrUpdateCertificates(); err != nil {
+			clusterLoggingRequest.Cluster.Status.Conditions.SetCondition(
+				condInvalid("Unable to create or update certificates"))
```

Comment:
@alanconway I don't think this is entirely accurate.  This is a hard failure for the logstore but if we have collection spec'd outside of managed storage... The operator would be degraded providing partial use.

Reply:
+1. We should also mark all the conditions for all pipelines to "default" as invalid if the are not already.  
If that leaves 0 valid pipelines, then this condition is Invalid, otherwise it is Degraded.

Reply:
ok